I cannot import GroupMember entities

### Expected behavior
I would expect that attached file should upload without any problems
Without GroupMember tab it uploads fine.

### Observed behavior
I'm getting a "java.lang.reflect.InvocationTargetException" error.

[userTest (1).xlsx](https://github.com/molgenis/molgenis/files/919938/userTest.1.xlsx)

Stacktrace:
```
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.molgenis.data.AbstractSystemEntityFactory.create(AbstractSystemEntityFactory.java:100)
	at org.molgenis.data.EntityReferenceCreatorImpl.getReference(EntityReferenceCreatorImpl.java:34)
	at org.molgenis.data.EntityManagerImpl.getReference(EntityManagerImpl.java:98)
	at org.molgenis.data.importer.emx.ImportWriter.toEntity(ImportWriter.java:305)
	at org.molgenis.data.importer.emx.ImportWriter.lambda$importData$5(ImportWriter.java:243)
	at com.google.common.collect.Iterators$8.transform(Iterators.java:794)
	at com.google.common.collect.TransformedIterator.next(TransformedIterator.java:48)
	at java.util.Spliterators$IteratorSpliterator.tryAdvance(Spliterators.java:1812)
	at java.util.stream.StreamSpliterators$WrappingSpliterator.lambda$initPartialTraversalState$0(StreamSpliterators.java:294)
	at java.util.stream.StreamSpliterators$AbstractWrappingSpliterator.fillBuffer(StreamSpliterators.java:206)
	at java.util.stream.StreamSpliterators$AbstractWrappingSpliterator.doAdvance(StreamSpliterators.java:161)
	at java.util.stream.StreamSpliterators$WrappingSpliterator.tryAdvance(StreamSpliterators.java:300)
	at java.util.Spliterators$1Adapter.hasNext(Spliterators.java:681)
	at com.google.common.collect.Iterators$6.hasNext(Iterators.java:612)
	at java.util.Iterator.forEachRemaining(Iterator.java:115)
	at org.molgenis.data.postgresql.PostgreSqlRepository.addBatching(PostgreSqlRepository.java:432)
	at org.molgenis.data.postgresql.PostgreSqlRepository.add(PostgreSqlRepository.java:256)
	at org.molgenis.data.AbstractRepositoryDecorator.add(AbstractRepositoryDecorator.java:161)
	at org.molgenis.data.cache.l1.L1CacheRepositoryDecorator.add(L1CacheRepositoryDecorator.java:63)
	at org.molgenis.data.AbstractRepositoryDecorator.add(AbstractRepositoryDecorator.java:161)
	at org.molgenis.data.AbstractRepositoryDecorator.add(AbstractRepositoryDecorator.java:161)
	at org.molgenis.data.index.IndexActionRepositoryDecorator.add(IndexActionRepositoryDecorator.java:90)
	at org.molgenis.data.AbstractRepositoryDecorator.add(AbstractRepositoryDecorator.java:161)
	at org.molgenis.data.AbstractRepositoryDecorator.add(AbstractRepositoryDecorator.java:161)
	at org.molgenis.data.AbstractRepositoryDecorator.add(AbstractRepositoryDecorator.java:161)
	at org.molgenis.data.validation.RepositoryValidationDecorator.add(RepositoryValidationDecorator.java:86)
	at org.molgenis.data.AbstractRepositoryDecorator.add(AbstractRepositoryDecorator.java:161)
	at org.molgenis.data.RepositorySecurityDecorator.add(RepositorySecurityDecorator.java:192)
	at org.molgenis.data.transaction.TransactionalRepositoryDecorator.lambda$add$19(TransactionalRepositoryDecorator.java:198)
	at org.springframework.transaction.support.TransactionTemplate.execute(TransactionTemplate.java:133)
	at org.molgenis.data.transaction.TransactionalRepositoryDecorator.add(TransactionalRepositoryDecorator.java:198)
	at org.molgenis.data.AbstractRepositoryDecorator.add(AbstractRepositoryDecorator.java:161)
	at org.molgenis.data.importer.emx.ImportWriter.update(ImportWriter.java:466)
	at org.molgenis.data.importer.emx.ImportWriter.importData(ImportWriter.java:244)
	at org.molgenis.data.importer.emx.ImportWriter.doImport(ImportWriter.java:92)
	at org.molgenis.data.importer.emx.ImportWriter$$FastClassBySpringCGLIB$$df3d0c02.invoke(<generated>)
	at org.springframework.cglib.proxy.MethodProxy.invoke(MethodProxy.java:204)
	at org.springframework.aop.framework.CglibAopProxy$CglibMethodInvocation.invokeJoinpoint(CglibAopProxy.java:721)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:157)
	at org.springframework.transaction.interceptor.TransactionInterceptor$1.proceedWithInvocation(TransactionInterceptor.java:99)
	at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:282)
	at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:96)
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:179)
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:656)
	at org.molgenis.data.importer.emx.ImportWriter$$EnhancerBySpringCGLIB$$da71707a.doImport(<generated>)
	at org.molgenis.data.importer.emx.EmxImportService.doImport(EmxImportService.java:83)
	at org.molgenis.data.importer.emx.EmxImportService.doImport(EmxImportService.java:70)
	at org.molgenis.data.importer.ImportJob.run(ImportJob.java:50)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.molgenis.data.UnknownEntityException: entity [sys_sec_Group] with id [editor] does not exist
	at org.molgenis.data.support.LazyEntity.getLazyLoadedEntity(LazyEntity.java:183)
	at org.molgenis.data.support.LazyEntity.set(LazyEntity.java:165)
	at org.molgenis.data.support.StaticEntity.set(StaticEntity.java:151)
	at org.molgenis.auth.Group.setActive(Group.java:38)
	at org.molgenis.auth.Group.setDefaultValues(Group.java:63)
	at org.molgenis.auth.Group.<init>(Group.java:14)
	... 55 more
```

--- a/molgenis-data/src/main/java/org/molgenis/auth/Group.java
+++ b/molgenis-data/src/main/java/org/molgenis/auth/Group.java
@@ -11,7 +11,6 @@ public class Group extends StaticEntity
 	public Group(Entity entity)
 	{
 		super(entity);
-		setDefaultValues();
 	}
 
 	public Group(EntityType entityType)

